### PR TITLE
 Got rid of toplevel messages

### PR DESCRIPTION
**Pull request by jasnow**
Got rid of "warning: class variable access from toplevel" messages.

### CODE REVIEW

File: lib/sorbet-rails/tasks/rails_rbi.rake [hunk 1]
```diff
@@ -67,7 +63,8 @@ namespace :rails_rbi do
   end
 
   def copy_bundled_rbi
-    bundled_rbi_path = File.join(@@sorbet_rails_rake_dir, "..", "rbi", ".")
+    sorbet_rails_rake_dir = "#{Dir.pwd}/lib/sorbet-rails/tasks"
```
Comment (manhhung741) [inline lib/sorbet-rails/tasks/rails_rbi.rake]: I'm not sure this work when people install the gem & run it from rake. `Dir.pwd` will refer to the location they run the rake task, not the root directory of the gem.

Comment (jasnow) [reply lib/sorbet-rails/tasks/rails_rbi.rake]: How about: sorbet_rails_rake_dir = "#{Gem::Specification.find_by_name("sorbet-rails").gem_dir}/lib/sorbet-rails/tasks"

Comment (jasnow) [reply lib/sorbet-rails/tasks/rails_rbi.rake]: Let's try.

File: lib/sorbet-rails/tasks/rails_rbi.rake [hunk 2]
```diff
@@ -3,10 +3,6 @@ require("sorbet-rails/routes_rbi_formatter")
 require("sorbet-rails/helper_rbi_formatter")
 require("sorbet-rails/utils")
 
-# this is ugly but it's a way to get the current directory of this script
-# maybe someone coming along will know a better way
-@@sorbet_rails_rake_dir = File.dirname(__FILE__)
```
Comment (palasha) [inline lib/sorbet-rails/tasks/rails_rbi.rake]: Could a simpler way to resolve this be to define an inline class with a constant?

eg:
```
class Foo
  RAKE_DIR = ...
end
```

And then reference `Foo::RAKE_DIR` at the callsite?

Comment (jasnow) [reply lib/sorbet-rails/tasks/rails_rbi.rake]: I will close this PR.

**Pull request closed without merging**